integration/terraform/ec2/README.md

```diff
@@ -130,8 +168,16 @@ See [docs](https://docs.aws.amazon.com/cli/latest/userguide/cli-services-ec2-key
 on creating the key pair.
 > Note: Store the private key in a secure location!
 
-**Reminder: the EC2 key pair must be in the same region as the instances, so this assumes that the key pair is created
-in the `us-west-2` region.**
+### Create ECR Repository
+|Field              |Value                   |
```

Comment:
**No blocker from me**: the EC2 README requires to create this resource beforehand. Therefore, I would say it would be more effortless to create these resources in terraform script and also let developer follow our requirements. For example: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecr_repository

Reply:
That should be addressed separately.